tcms/testruns/views.py

```diff
@@ -334,9 +334,10 @@ class TestRunReportView(TemplateView, TestCaseRunDataMixin):
     """Test Run report"""
 
     template_name = 'run/report.html'
+    run_id = None
 
-    def get(self, request, run_id):
-        self.run_id = run_id
+    def get(self, request, *args, **kwargs):
+        run_id = args[0]
```

Comment:
Why ? I don't understand what are you trying to achieve here ?

Reply:
The old parameters differ from the overriden method.

Reply:
ok, let's revert this change for now. modifying view method may have a significant impact and we need more time to investigate and re-test properly.